conda-forge.yml

```
@@ -4,3 +4,10 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+build_platform:
+  linux_aarch64: linux_aarch64
```

Are we able to cross-compile for ARM too?

Then we could use Azure for all builds

```suggestion
  linux_aarch64: linux_64
```

---

I think with this PR it becomes a simple recipe now so Travis is working fine, so I didn't bother to cross compile but either way is fine. No strong opinion. (Travis's ppc64le pipelines on the other hand are just not functioning.)

---

Personally would rather replace it

At this stage I'm unable to use Travis CI's web app at all. Have an open issue with them that has yet to be resolved after weeks

Would rather use cross-compilation for all of them if possible

Though please let me know if there is some reason to still use Travis here

---

I will do anything I can to accelerate merging this PR, so I am accepting with no objection 🙂